ethereum/block.go

```diff
@@ -124,3 +176,42 @@ func produceToKafka(kafkaProducer *sarama.SyncProducer) {
 		}
 	}
 }
+
+//Helper functions
+
+func ParseLiquidStakingStakeTokensEvent(vLog types.Log) (liquidStaking.LiquidStakingStakeTokens, error) {
+	var event liquidStaking.LiquidStakingStakeTokens
+	err := contracts.LiquidStaking.GetABI().UnpackIntoInterface(&event, EventStakeTokens, vLog.Data)
+	if err != nil {
+		return event, errors.New("failed to Unpack event StakeTokens")
+	}
+	event.Raw = vLog
+	event.AccountAddress = common.HexToAddress(vLog.Topics[1].Hex())
+	event.Tokens = vLog.Topics[2].Big()
+	event.FinalTokens = vLog.Topics[3].Big()
+
+	return event, nil
+}
+func ParseLiquidStakingUnstakeTokensEvent(vLog types.Log) (liquidStaking.LiquidStakingUnstakeTokens, error) {
+	var event liquidStaking.LiquidStakingUnstakeTokens
+	err := contracts.LiquidStaking.GetABI().UnpackIntoInterface(&event, EventUnstakeTokens, vLog.Data)
+	if err != nil {
+		return event, errors.New("failed to Unpack event StakeTokens")
+	}
+	event.Raw = vLog
+	event.AccountAddress = common.HexToAddress(vLog.Topics[1].Hex())
+	event.Tokens = vLog.Topics[2].Big()
+	event.FinalTokens = vLog.Topics[3].Big()
+	return event, nil
+}
+func ParseTokenWrapperWithdrawUTokensEvent(vLog types.Log) (tokenWrapper.TokenWrapperWithdrawUTokens, error) {
+	var event tokenWrapper.TokenWrapperWithdrawUTokens
+	err := contracts.TokenWrapper.GetABI().UnpackIntoInterface(&event, EventWithdrawUTokens, vLog.Data)
+	if err != nil {
+		return event, errors.New("failed to Unpack event StakeTokens")
```

Comment:
should be - failed to Unpack event WithdrawUTokens

Reply:
fixed